src/metabase/lib/breakout.cljc

```diff
@@ -84,10 +90,42 @@
                     pos (assoc :breakout-position pos)))
                cols))))))
 
+(mu/defn existing-breakout :- [:maybe ::lib.schema.ref/ref]
+  "Returns the existing breakout MBQL expression for `column` in a stage if there is one."
+  ([query stage-number column]
+   (existing-breakout query stage-number column nil))
+
+  ([query                                         :- ::lib.schema/query
+    stage-number                                  :- :int
+    column                                        :- ::lib.schema.metadata/column
+    {:keys [same-temporal-bucket?], :as _options} :- [:maybe
+                                                      [:map
+                                                       {:closed true}
```

Comment:
because if you're using other options it doesn't understand, like `:same-temporal-bucket` instead of `:same-temporal-bucket?`, it's a typo and it's not doing what you probably think it is?

Reply:
I generally assume functions don't barf if they get options they don't understand. It's not a big issue here, as the options are not passed further down the line, but I still find it unconventional.

Reply:
fixed